common/test/views/support/ContentFooterContainersLayoutTest.scala

```diff
@@ -0,0 +1,64 @@
+package views.support
+
+import conf.Switches.OutbrainSwitch
+import contentapi.FixtureTemplates
+import model.{Article, Content, RelatedContent}
+import org.scalatest.{FlatSpec, Matchers}
+import play.twirl.api.Html
+
+class ContentFooterContainersLayoutTest extends FlatSpec with Matchers {
+
+  OutbrainSwitch.switchOn()
+
+  private def contentItem(shouldHideAdverts: Boolean = false, commentable: Boolean = true): Content = {
+    new Article(FixtureTemplates.emptyApiContent.copy(fields = Some(Map(
+      "showInRelatedContent" -> "true",
+      "shouldHideAdverts" -> shouldHideAdverts.toString,
+      "commentable" -> commentable.toString
+    ))))
+  }
+
+  private def relatedContent(): RelatedContent = RelatedContent(Seq(contentItem()))
+
+  private def buildHtml(content: Content, related: RelatedContent, isAdFeature: Boolean = false): Html = {
+    ContentFooterContainersLayout(content, related, isAdFeature) {
+      Html("storyPackageHtml ")
+    } {
+      Html("onwardHtml ")
+    } {
+      Html("commentsHtml ")
+    } {
+      Html("mostPopularHtml ")
+    } {
+      Html("highRelevanceCommercialHtml ")
+    } {
+      Html("standardCommercialHtml ")
+    } {
+      Html("outbrainHtml ")
+    }
+  }
+
+  it should "show all footer containers in right order by default" in {
+    val html = buildHtml(contentItem(), relatedContent())
+    html.toString shouldBe
+      "highRelevanceCommercialHtml storyPackageHtml onwardHtml outbrainHtml commentsHtml mostPopularHtml " +
```

Comment:
outbrainHtml always needs to be on the second visible spot (that means third because highRelevance component is being served by JS so at this point we have to place it there at all time).

So the right sequence should be like this: highRelevanceCommercialHtml storyPackageHtml outbrainHtml onwardHtml commentsHtml mostPopularHtml " + "standardCommercialHtml "